fetch/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Programmer Humour</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <h1>XKCD Comic Viewer</h1>
+    <div id="comic-container">  
+        <img id="comic-img" alt="XKCD Comic" />
```

Comment:
As a user, it's a little weird that when I first open the page it shows me a broken image... Can you think of a more natural effect to show users when they first load the page?